core/src/utils/mf-helper.ts

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+
+export const initModuleFederationModule = async (scope: string, module: string) => {
+  // Initializes the share scope. This fills it with known provided modules from this build and all remotes
+  // @ts-ignore webpack built in function
+  await __webpack_init_sharing__('default');
+  // @ts-ignore dynamic scope
+  const container = window[scope]; // or get the container somewhere else
+  // Initialize the container, it may provide shared modules
+  // @ts-ignore webpack built in scope
+  await container.init(__webpack_share_scopes__.default);
+  // @ts-ignore dynamic scope
+  const factory = await window[scope].get(module);
+  const Module = factory();
+  return Module;
```

Comment:
These logic is in `entry.js`, why should we execute by selves ?
Will throw error in vite ?

Reply:
load the mf_xx.js just create one Obj under window， the initialization should be executed by ourselves.

In vite, it wont run into this function, because vite can't load mf script.